libs/spark/src/NavBarButton.tsx

```diff
@@ -3,16 +3,22 @@ import { withStyles } from '@material-ui/core/styles';
 import { Button, ButtonProps } from '@material-ui/core';
 import { theme } from './styles';
 
-export type NavBarButtonProps = Omit<ButtonProps, 'variant' | 'color' | 'size'>;
+export type NavBarButtonProps = Omit<ButtonProps, 'variant' | 'color'>;
```

Comment:
Unless we can figure out a way to associate this with the declaration of `NavBarButton` then its useless as anything except a reference.